app/admin/user.rb

```diff
@@ -167,6 +167,18 @@
         end
       end
 
+      tab I18n.t('experience') do
+        panel I18n.t('user_open_source') do
+          table_for user.contributions.valid_pull_requests.order(created_at: :desc), i18n: Contribution, id: 'table_admin_contributions' do
```

Comment:
Why the need for assigning a custom id?

Reply:
Removed this argument, there was no need for it.